cypress/integration/Internationalization.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Internationalization
+  As a user
```

Comment:
And for the positioning. I think about putting the language switch in the user menu and or the footer to not clutter the toolbar with it.

Reply:
Yes, it just updates the feature description. But the feature description is important, after all it tells you for whom this feature creates value.

Reply:
Regarding the positioning of the language chooser: We should complement the language menu with links in the footer! Read [this](http://www.flagsarenotlanguages.com/blog/).

I scrolled down to the conclusion and I slightly disagree with the globe icon instead of the country flag. I'm OK to offend Austrians and German-speaking Suisse people as long as they can immediately find the language chooser menu. To me the globe icon is simply too vague.

But I agree and I can see the point in having language links in the footer.